src/pages/TakeTurnPage.vue

```diff
@@ -93,10 +94,21 @@ const onBodegasGetted = (bodegas: Bodega[]) => {
   bodegaSelect.value.options.length = 0;
   bodegaSelect.value.selected = null;
   bodegaSelect.value.options.push(...bodegas);
-  step.value++;
+  if (bodegaSelect.value.options.length == 1) {
+    warehouseCant.value = 1;
+    bodegaSelect.value.selected = bodegas[0];
+    step.value = 3;
+  } else {
+    warehouseCant.value = bodegaSelect.value.options.length;
+    step.value++;
+  }
 };
 
 const moveStepper = (type: 'next' | 'previous') => {
-  type === 'next' ? step.value++ : step.value--;
+  type === 'next'
+    ? step.value++
+    : type === 'previous' && warehouseCant.value == 1
```

Comment:
When a ternary operator has more than 1 evaluation, you should think in change it to a 'if' with body, and return statement